app/controllers/commits_controller.rb

```diff
@@ -26,4 +26,8 @@ def session_votes_by_commit_id(commits)
       [v.commit_id, v]
     end
   end
+
+  def scope
+    Commit::SCOPES[params[:scope]]
```

Comment:
Une mini suggestion qui améliore l'erreur le jour où on ajoute un nouveau scope : 

```suggestion
    Commit::SCOPES.fetch(params[:scope])
```

Ça donnera une erreur ici du genre `KeyError: key not found: "best"` plutôt que `TypeError: nil is not a symbol nor a string` (car on s'en sert dans `public_send`).

Reply:
Excellent ! Cette erreur m'agaçait justement :)